### PR TITLE
Upgrade `protobuf` to enable publishing `aesm-client`

### PR DESCRIPTION
**Pull request by raoulstrackx**

### CODE REVIEW

File: aesm-client/Cargo.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ sgx-isa = { version = "0.3.0", path = "../sgx-isa"}
 # External dependencies
 byteorder = "1.0"          # Unlicense/MIT
 lazy_static = "1"          # MIT/Apache-2.0
-protobuf = "2.8.0"         # MIT/Apache-2.0
+protobuf = "2.22.0"        # MIT/Apache-2.0
```
Comment (jethrogb) [inline aesm-client/Cargo.toml:35]: Why are you updating the version requirement? AFAICT Message::parse_from_bytes already existed in 2.8.

Comment (raoulstrackx) [reply aesm-client/Cargo.toml:35]: I don't see it: https://docs.rs/protobuf/2.18.1/protobuf/fn.parse_from_bytes.html Am I missing something?

Comment (jethrogb) [reply aesm-client/Cargo.toml:35]: Sorry, you're right, I didn't look correctly. The minimum version should be 2.19.0.

**Pull request merged**